src/pages/overview/pdf-services-api/gettingstarted.md

```diff
@@ -183,7 +183,7 @@ After downloading the zip, you can run the samples in the zip directly by settin
 
 #### Setup a Java environment
 
-1.  Install [Java 8 or above](http://www.oracle.com/technetwork/java/javase/downloads/index.html).
+1.  Install [Java 11 or above](http://www.oracle.com/technetwork/java/javase/downloads/index.html).
```

Comment:
Do we mention on this page in the SDK section ( that these whitelisting only valid for the default case or internal assets case ) ?

Reply:
https://github.com/AdobeDocs/pdfservices-api-documentation/pull/744#discussion_r1409403440